src/QrCode.module.css

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2024 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.qrCode img {
+  max-width: 100%;
+  image-rendering: pixelated;
```

Comment:
I think giving the qr code image a border radius would make it look more at home.

```suggestion
  image-rendering: pixelated;
  border-radius: var(--cpd-space-4x);
```

![Screenshot from 2024-09-02 16-53-51](https://github.com/user-attachments/assets/93e8e857-1c60-49f9-9e3e-5807e9a82c59)

Reply:
Oh nice! 👍